### PR TITLE
update

### PR DESCRIPTION
**Pull request by bobzoller**
pull upstream, resolve conflicts.

### CODE REVIEW

File: macros/audit.sql [hunk 1]
```diff
@@ -1,37 +1,54 @@
...
         invocation_id
-        )
+    )
 
     values (
         '{{ event_name }}',
-        convert_timezone('UTC', current_timestamp)::timestamp_ntz,
-        {% if variable != None %}'{{ schema }}'{% else %}null::varchar(512){% endif %},
-        {% if variable != None %}'{{ relation }}'{% else %}null::varchar(512){% endif %},
+        {{ dbt_utils.current_timestamp_in_utc() }},
```
Comment (dwallace0723) [inline macros/audit.sql]: @bobzoller so this is what we actually just got rid of as per Snowflake's recommendation (redundant casting)

Comment (bobzoller) [reply macros/audit.sql]: ah k

File: macros/audit.sql [hunk 2]
```diff
@@ -1,37 +1,54 @@
...
+
+{% endmacro %}
+
 {% macro get_audit_relation() %}
+    {%- set audit_schema=logging.get_audit_schema() -%}
+
     {%- set audit_table =
         api.Relation.create(
+            database=target.database,
+            schema=audit_schema,
```
Comment (dwallace0723) [inline macros/audit.sql:18]: @bobzoller I think we want these to remain static

Comment (bobzoller) [reply macros/audit.sql:18]: before I use this PR in production, I'll be adding this macro to legacy, which (if I'm grokking this codebase right) will make the schema static:
```
{% macro get_audit_schema() %}
  {{ return('dbt_meta') }}
{% endmacro %
```
seem right?

Comment (bobzoller) [reply macros/audit.sql:18]: https://github.com/fishtown-analytics/dbt-event-logging#customising-audit-schema

**Review by dwallace0723 — APPROVED** [review]

**Pull request merged**